src/commands/Giveaway/greroll.ts

```diff
@@ -16,26 +16,22 @@ import { FetchError } from 'node-fetch';
 	aliases: ['gr', 'groll'],
 	description: LanguageKeys.Commands.Giveaway.GiveawayRerollDescription,
 	extendedHelp: LanguageKeys.Commands.Giveaway.GiveawayRerollExtended,
-	requiredPermissions: ['READ_MESSAGE_HISTORY'],
-	runIn: ['text'],
-	usage: '[winners:number{1,100}] [message:message]',
-	usageDelim: ' '
+	permissions: ['READ_MESSAGE_HISTORY'],
+	runIn: ['text']
 })
-export default class extends SkyraCommand {
+export class UserCommand extends SkyraCommand {
 	// eslint-disable-next-line @typescript-eslint/explicit-member-accessibility
 	#kResolvedEmoji = resolveEmoji(kRawEmoji)!;
 
-	public async run(message: GuildMessage, [winnerAmount = 1, rawTarget]: [number, GuildMessage | undefined]) {
-		const t = await message.fetchT();
-		const target = await this.resolveMessage(message, rawTarget, t);
+	public async run(message: GuildMessage, args: SkyraCommand.Args) {
+		const winnerAmount = args.finished ? 1 : await args.pick('integer').catch(() => 1);
+		const rawTarget = args.finished ? undefined : cast<GuildMessage>(await args.pick('message'));
+		const target = await this.resolveMessage(message, rawTarget, args.t);
 		const { title } = target.embeds[0];
 		const winners = await this.pickWinners(target, winnerAmount);
 		const content = winners
-			? t(LanguageKeys.Giveaway.EndedMessage, {
-					winners: winners.map((winner) => `<@${winner}>`),
-					title: title!
-			  })
-			: t(LanguageKeys.Giveaway.EndedMessageNoWinner, { title: title! });
+			? args.t(LanguageKeys.Giveaway.EndedMessage, { winners: winners.map((winner) => `<@${winner}>`), title: title! })
```

Comment:
While a small nit, this is needed for clients to fetch nicknames or something

```suggestion
			? args.t(LanguageKeys.Giveaway.EndedMessage, { winners: winners.map((winner) => `<@!${winner}>`), title: title! })
```